spec/controllers/state_file/questions/verification_code_controller_spec.rb

```diff
@@ -85,6 +85,8 @@
       let(:intake) do
         build(:state_file_az_intake, contact_preference: "email", email_address: "someone@example.com", visitor_id: "v1s1t1n9").tap do |intake|
           intake.raw_direct_file_data = nil
+          unused_id = ([StateFileAzIntake.last&.id, StateFileIdIntake.last&.id].compact.max || 0) + 1
+          intake.id = unused_id
```

Comment:
just want to grab an id that won't clash with any existing ones on either AZ/ID intakes table